torch/linalg/__init__.py

```diff
@@ -339,6 +339,88 @@
     (tensor(3.7417), tensor(11.2250))
 """)
 
+cond = _add_docstr(_linalg.linalg_cond, r"""
+linalg.cond(input, p=None, *, out=None) -> Tensor
+
+Computes the condition number of a matrix :attr:`input`,
+or of each matrix in a batched :attr:`input`, using the matrix norm defined by :attr:`p`.
+For norms ``p = {'fro', 'nuc', inf, -inf, 1, -1}`` this is defined as the matrix norm of :attr:`input`
+times the matrix norm of the inverse of :attr:`input`. And for norms ``p = {None, 2, -2}`` this is defined as
+the ratio between the largest and smallest singular values.
+
+This function supports ``float``, ``double``, and only on CPU, ``cfloat`` and ``cdouble`` dtypes for :attr:`input`.
+
+.. note:: For ``p = {None, 2, -2}`` the condition number is computed as the ratio between the largest and smallest singular values
+          computed using :func:`torch.linalg.svd`. For these norms :attr:`input` may be a non-square matrix or batch of non-square matrices.
+          For other norms, however, :attr:`input` must be a square matrix or a batch of square matrices,
+          and if this requirement is not satisfied a RuntimeError will be thrown.
+
+.. note:: If :attr:`input` is a non-invertible matrix then a tensor containing infinity will be returned.
```

Comment:
Darn, I just realized this note will need to be updated, too, since non-invertbility doesn't apply to both cases, and the next sentence is redundant with the previous note.

Sorry I missed that on the last pass, @IvanYashchuk.

Reply:
I updated this note mentioning that it applies only for some `p`.
```
.. note:: For ``p = {'fro', 'nuc', inf, -inf, 1, -1}`` if :attr:`input` is a non-invertible matrix then
          a tensor containing infinity will be returned. If :attr:`input` is a batch of matrices and one
          or more of them is not invertible then a RuntimeError will be thrown.
```